ui/roll20_character_sheet_0.2.html

```diff
@@ -2184,6 +2186,34 @@
   on("remove:repeating_innate change:repeating_innate:skillCP",
     function () { updateTotalCP("innate", "skill") });
 
+  function getAttrsAfterUiUpdates(attributes, callback) {
+    setTimeout(() => {
+      getAttrs(attributes, callback);
+    }, 0);
+  }
```

Comment:
This may work. We can use `setTimeout` to delay the function call to be after the user blurs from a field and all other listeners have gone through the event queue.